File: Week-08/Md Sohel/Which Mixture.java
/* package codechef; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Scanner sc = new Scanner(System.in);
		int t, a, b;
		
		t = sc.nextInt();
		
		while(t > 0){
		    a = sc.nextInt();
		    b = sc.nextInt();
		    
		    if(a > 0 && b > 0){
		        System.out.println("Solution");
		    }else if (a == 0){
		        System.out.println("Liquid");
		    }else {
		        System.out.println("Solid");
		    }
		    t--;
		}
	}
}

File: Week-08/Afzal Sakib/Which Mixture/Which_Mixture.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
     int t,i,j;
     cin>>t;
    vector<vector<int>>num(t,vector<int>(2));

   for(i=0; i<t; i++)
   {
    for(j=0; j<2; j++)
    cin>>num[i][j];
   }

   for(i=0; i<t ;i++)
   {
       if(num[i][0]>0 && num[i][1]>0)
    cout<<"Solution\n";
    else if(num[i][0]==0 )
    cout<<"Liquid\n";
    else if( num[i][1]==0)
    cout<<"Solid\n";
   }

  return 0;
}